web/src/list_widget.ts

```diff
@@ -261,6 +262,13 @@ export function create<Key, Item = Key>(
         old_widget.clear_event_handlers();
     }
 
+    // When `html` is the scroll container, scroll event is fired on `window`.
+    // TODO: Add an online reference for this since it's not intuitive.
```

Comment:
I don't understand the proposal in this comment.

Reply:
Renamed `$scroll_event_handler` to `$scroll_listening_element`.


Modified the comment to be more clear:

```
    // When `html` is is `$scroll_container`, scroll event is fired on `window/document`.
    // This is because when a scroll event happens on the body or html element,
    // the event is fired on the document object, not the body or html element itself.
    // We still keep `html` as `$scroll_container` to use it's various methods as `HTMLElement`.
```

Reply:
Simplified and moved this inside the conditional block:

```js
    if ($scroll_listening_element.is("html")) {                                      
        // When `$scroll_container` is the entire page (`html`),                     
        // scroll events are fired on `window/document`, so we need to               
        // listen for scrolling events on that.                                      
        //                                                                           
        // We still keep `html` as `$scroll_container` to use                        
        // its various methods as `HTMLElement`.                                     
        $scroll_listening_element = $(window);                                       
    }                                                                                
```